setup.py

```diff
@@ -48,6 +48,7 @@
         'pynio',
         'pyproj>=2.1'
         'pyyaml',
+        'progressbar2',
```

Comment:
If this is needed, you will also need to add it to package/meta.yml and possibly doc/requirements.txt